%YAML:1.0
CameraInfo:
  -
    Name: usbcam1fdemo1
    DevID: "1"
    Width: 320
    Height: 240
    PixelFormat: MJPG
    NRotate90: 0
    Rectification: 1
    Alpha: 0.8
    K: !!opencv-matrix
      rows: 3
      cols: 3
      dt: d
      data: [ 4.1249332581400529e+02, 0., 1.1034980325175262e+02, 0.,
          4.1249332581400529e+02, 1.2088269510065035e+02, 0., 0., 1. ]
    D: !!opencv-matrix
      rows: 1
      cols: 5
      dt: d
      data: [ -2.4585340187798215e+00, 5.0602025619665234e-00, 0., 0., 0. ]
    R: !!opencv-matrix
      rows: 3
      cols: 3
      dt: d
      data: [ 1.0, 0.0, 0.0,  0.0, 1.0, 0.0,  0.0, 0.0, 1.0 ]

BlobTracker2:
  -
    # SBDParams_filterByColor: 0
    # SBDParams_blobColor: 0
    # SBDParams_minThreshold : 30
    # SBDParams_maxThreshold : 200
    # SBDParams_filterByArea : true
    # SBDParams_minArea : 40
    # SBDParams_filterByCircularity : true
    # SBDParams_minCircularity : 0.10
    # SBDParams_filterByConvexity : true
    # SBDParams_minConvexity : 0.87
    # SBDParams_filterByInertia : true
    # SBDParams_minInertiaRatio : 0.01
    ThreshH: 180
    ThreshS: 255
    ThreshV: 40
    NDilate1: 0  #2
    NErode1: 0  #2
    SWidth: 30
    NonZeroMin: 0.3  #0.5
    NonZeroMax: 3.0  #1.5
    VPMax: 5.0
    VSMax: 1.0
    NReset: 3
    DistMaxCalib: 0.8
    DSMaxCalib: 0.5
    DSEmp: 4.0
    DPEmp: 10.0
    NCalibPoints: 10

ObjDetTrack:
  -
    Width: 320
    Height: 240
    ThreshBlkH: 180
    ThreshBlkS: 255
    ThreshBlkV: 40
    BS_History: 30.0
    BS_VarThreshold: 10.0
    BS_DetectShadows: true
    NErode1: 0
    BinsH: 100
    BinsS: 10
    Fbg: 1.5
    Fgain: 0.5
    NumModel: 5
    NumFramesInHist: 200
    NThreshold2: 150
    NErode2: 2
    NDilate2: 7
    NCalibBGFrames: 3
    ObjSW: 3
    ObjSH: 3
    MvSW: 3
    MvSH: 3

StereoInfo:{}
